cmsmon-tools.spec

```diff
@@ -0,0 +1,72 @@
+### RPM cms cmsmon-tools 0.3.10
+## INITENV +PATH PYTHONPATH %i/${PYTHON_LIB_SITE_PACKAGES}
+## NOCOMPILER
+
+%define pkg CMSMonitoring
+%define ver %realversion
+Source0: https://github.com/dmwm/%pkg/archive/%ver.tar.gz
```

Comment:
define commands here once e.g.
```
%define cmsmon_commands nats-sub nats-pub nats-exitcodes-termui dbs_vm
```
and then use %{cmsmon_commands} where needed

Reply:
done